doc/adr/0007-use-roles-to-authorise-users-to-use-data.md

```diff
@@ -0,0 +1,39 @@
+# 7. Use roles to authorise users to use certain data
+
+Date: 2020-04-23
+
+## Status
+
+Accepted
+
+## Context
+
+We need a way to authorise users in the MOLGENIS "Armadillo" service to use data. We want this to be as straightforward as possible.
+
+Other techniques are ACL's for fine grained permissions on resources and ID-based. 
+
+**ACL's**
+It is more time consuming to implement ACL's. With roles the you are less flexible, but we believe this is sufficient to use DataSHIELD.
+
+**ID-based**
+The ID-based approach was suggested as a first solution to make the service secure. A big disadvantage is that anyone with the link can 
+share the data. That means that there has to be a lot of trust between the data manager and the researcher. In practices, this is not feasible.
+
+## Decision
+
+We will use roles as a basic principle to give users permission on the data they need to have access to. 
+Within the MOLGENIS "Armadillo" service there are at least these 2 types of roles:
+- Researcher
```

Comment:
I expect that in time we will need 'role per study'. Do you agree? 
So how will that play out? Is that then implemented by 'read' on a particular folder?

Reply:
That is correct. It should match the role descriptions below, but indeed you can have for instance 10 study folders with subsets of the data in it and apply read rights for different researchers at a folder level.